.github/workflows/helm-release.yml

```diff
@@ -39,14 +39,13 @@ jobs:
             -f target_commitish="${GITHUB_REF_NAME}" \
             -f previous_tag_name="${{ steps.previous-version.outputs.tag }}" | jq -r .body > charts/cryostat/release-notes.md
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.5.0
+        uses: helm/chart-releaser-action@v1.6.0
```

Comment:
I think this change should be undone and split out into a separate PR (against `main`)

Reply:
ooh wait, I dint make this change!

Reply:
Not sure how it snuck in here then - either way, please remove it from this PR.

Reply:
It came from Thuan's commit